README.md

```diff
@@ -62,6 +62,33 @@ hzc --help
 # Non-interactive mode
 hzc map --name myMap put --key myKey --value myValue
 ```
+### Keyboard Shortcuts
+Hazelcast CLC inherits following shortcuts from [c-bata/go-prompt](https://github.com/c-bata/go-prompt)
```

Comment:
I think there's no need to add a reference to that link for the shortcuts. Let's add all of the supported shortcuts in a separate PR.